src/ontology/uberon-edit.obo

```diff
@@ -222010,6 +222010,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500121
+name: paroccipital process
+def: "An occasional process of bone extending downward from the jugular process of the occipital bone." [https://medical-dictionary.thefreedictionary.com/paroccipital+process]
+synonym: "paramastoid process" EXACT []
```

Comment:
Consider additional synonyms. From https://doi.org/10.1016/j.wneu.2018.06.056:

"The paramastoid process (also termed the paraoccipital, paracondylar, parajugular, and/or estiloid process)[3](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib3), [4](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib4) ... is a bony excrescence, which is an inferior extension of the jugular process of the occipital bone.[3](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib3), [5](https://www.sciencedirect.com/science/article/pii/S1878875018312713?via%3Dihub#bib5)"